lib/config.mli

```diff
@@ -109,6 +109,7 @@ type config = {
   cgi : bool;
   forced_plugins : string list;
   plugins : string list;
+  preferred_countries : string list option;
```

Comment:
do we really want a None instead of empty list if there is no preferred_countries set?